extensions/npm/main.ts

```diff
@@ -190,7 +190,7 @@ async function checkDryRun(subcommand: string, args: string[]) {
     return;
   }
 
-  const jobId = await PhylumApi.analyze("npm", lockfile.packages);
+  const jobId = await PhylumApi.analyze(undefined, lockfile.packages);
```

Comment:
Just to confirm: Did you test any/all of the existing extensions without making any changes to confirm this isn't a breaking change (to us at least)?

Reply:
Yes. I have confirmed that the extensions work without modification.